ros2_socketcan_msgs/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ find_package(ament_cmake_auto REQUIRED)
 ament_auto_find_build_dependencies()
 
 rosidl_generate_interfaces(${PROJECT_NAME}
-  "msg/Frame.msg"
+  "msg/FdFrame.msg"
```

Comment:
@kenji-miyake it also changes the `msg/Frame.msg` so this will be a breaking change for the users.

Reply:
I thought it's not used yet, but I'll check it later.

Reply:
But it should be a relatively easy fix for the users to adapt.

Reply:
Oh I've read the main issue again,
> Use `can_msgs/msg/Frame` for traditional CAN and a new message for CAN FD

It was my mistake, It seems ok.